tutorials/nservicebus-101/lesson-3/tutorial.md

```diff
@@ -0,0 +1,203 @@
+ 1. On the right, select the **Multiple startup projects** radio button.
+ 1. In the list box's **Action** column, set the value to **Start** for the **ClientUI** and **Sales** projects. (It doesn't make sense to do so for **Messages**, as it is just a class library and can't be started directly.)
+
+If you run the project now, you'll find that both ClientUI and Sales will start up. ClientUI will work just as it did before, and Sales will start up and wait for messages that will never arrive.
+
+
+### Moving the handler 
+
+Now let's move the handler from ClientUI over to Sales where it belongs.
+
+ 1. In the Solution Explorer, find **PlaceOrderHandler.cs** in the **ClientUI** project, and drag it to the **Sales** project.
+ 1. Open the new **PlaceOrderHandler.cs** in **Sales** and change the namespace from `ClientUI` to `Sales` to match its new home.
+ 1. Visual Studio's default action when you drag files between projects is to copy them, so you must delete the old **PlaceOrderHandler.cs** from the **ClientUI** endpoint.
+
+So now that the handler is in the correct endpoint, what would happen if we started the solution? Sales now has a message handler, but recall that ClientUI is still calling `endpointInstance.SendLocal(command)` which effectively sends the message to itself, but it doesn't have a handler anymore.
+
+If you attempt to place an order in the ClientUI, an exception will be thrown because ClientUI no longer has a handler for it:
+
+WARNING: System.InvalidOperationException: No handlers could be found for message type: Messages.Commands.PlaceOrder
+
+In fact, you will probably get a giant wall of exception text, because the message is tried and retried, and then retried some more after successively longer delays, until finally failing for good some time later. We'll cover this behavior in more detail in [Lesson 5: Retrying errors](../lesson-5/).
+
+The important part is, this is a good thing! If we accidentally send a message to an endpoint we didn't intend, it won't just fail silently, lost forever to the nothingness of the great beyond.
+
+
+### Sending to another endpoint
+
+Now we need to fix up the ClientUI so that it is sending `PlaceOrder` to the Sales endpoint.
+
+ 1. In the **ClientUI** endpoint, modify the **Program.cs** file so that `endpointInstance.SendLocal(command)` is replaced by `endpointInstance.Send(command)`.
+ 1. In the `AsyncMain` method of the same file, change the call to use the MSMQ transport to access the routing configuration and specify the logical routing for `PlaceOrder` by changing your code as follows.
+
+snippet:AddingRouting
+
+This establishes that commands of type `PlaceOrder` should be sent to the **Sales** endpoint. While these two calls could be strung together in the same statement, there will commonly be multiple logical routes specified, so it's advantageous to assign the `routing` variable for future use.
```

Comment:
>  While these two calls could be strung together in the same statement, there will commonly be multiple logical routes specified, so it's advantageous to assign the `routing` variable for future use.

remove this. the choice of not fluent or fluent in this context does not justify the text